src/Cell.tsx

```diff
@@ -1,5 +1,5 @@
 import { memo } from 'react';
-import { css } from '@linaria/core';
+import { css } from '@wyw-in-js/core';
```

Comment:
Can't we keep these imports as `@linaria/core`? The package still exists

Reply:
yeah, was just fixing it